packages/competence/INSTALL.md

```
@@ -271,13 +271,84 @@ docker run -d --name competence \
+- **Cold start of roughly 5–15 seconds** for the first request after idle.
+- **The service URL is a configuration dependency.** It is baked into
+  `TI_WEB_TRUSTED_ORIGINS` and the OAuth callback; recreating the service changes
+  it and breaks sign-in until `deploy.sh` is re-run and the redirect URI updated.
+- **Test-user mode is on**, so any authenticated visitor can act as any
+  employee. IAP is the only thing preventing that. Do not disable it, and do not
+  put real employee data here.
+- **Locked out?** `local` auth is disabled, so a broken OAuth client locks
+  everyone out of the app. Re-enable it temporarily — safe, because IAP still
+  fronts the service (the `^:^` prefix changes the delimiter so the comma is part
+  of the value):
+  ```bash
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)